contents/docs/cdp/common-questions.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Troubleshooting and FAQs
```

Comment:
could we add the common question around 
- missing events - @MarconLP  had that doc somewhere - how to file bug reports for those. 

Marcus might also have more common questions and FAQs